packages/rollup-plugin-html/test/rollup-plugin-html.test.ts

```diff
@@ -421,6 +421,22 @@ describe('rollup-plugin-html', () => {
     expect(pageC).to.include('<script type="module" src="./shared.js"></script>');
   });
 
+  it('can exclude globs', async () => {
+    const config = {
+      plugins: [
+        rollupPluginHTML({
+          input: 'exclude/**/*.html',
+          exclude: '**/partial.html',
+          rootDir,
+        }),
+      ],
+    };
+
+    const bundle = await rollup(config);
+    const { output } = await bundle.generate(outputConfig);
+    expect(output.length).to.equal(2);
```

Comment:
why is this not 1? 
only `packages/rollup-plugin-html/test/fixtures/rollup-plugin-html/exclude/index.html` should be considered right? 🤔

Reply:
```js
[
  {
    exports: [ 'default' ],
    facadeModuleId: '@web/rollup-plugin-html-noop',
    isDynamicEntry: false,
    isEntry: true,
    isImplicitEntry: false,
    modules: [Object: null prototype] {
      '@web/rollup-plugin-html-noop': [Object]
    },
    name: [Getter],
    type: 'chunk',
    code: 'var rollupPluginHtmlNoop = "noop";\n' +
      '\n' +
      'export default rollupPluginHtmlNoop;\n',
    dynamicImports: [],
    fileName: 'rollup-plugin-html-noop.js',
    implicitlyLoadedBefore: [],
    importedBindings: {},
    imports: [],
    map: null,
    referencedFiles: []
  },
  {
    fileName: 'index.html',
    name: 'index.html',
    isAsset: [Getter],
    source: '<html><head></head><body><a href="assets/docs/partial.html"></a>\n' +
      '</body></html>',
    type: 'asset'
  }
]
```

Reply:
does that explanation satisfy, @daKmoR ? should I make it explicit in the test code?